### PR TITLE
Restructured tests

### PR DESCRIPTION
**Pull request by ljaljushkin**
- extracted common test helpers
- separated tests related to range-init

### CODE REVIEW

File: tests/test_helpers.py [hunk 1]
```diff
@@ -10,25 +10,23 @@
  See the License for the specific language governing permissions and
```
Comment (vshampor) [inline tests/test_helpers.py:10]: I think that some of the logic of Pytest's test case collection depends on the files containing test cases having names starting with `test_`. If you intend this file not to contain any actual test cases to be run, perhaps we should rename it to simply `tests\helpers.py`

Comment (ljaljushkin) [reply tests/test_helpers.py:10]: there're still self-check tests for helper: e.g. `test_basic_model_is_valid`, `test_basic_model_has_expected_params`

Comment (ljaljushkin) [reply tests/test_helpers.py:10]: eventually, decided to separate helpers for tests from actual test functions

**Comment by ljaljushkin** [comment]
Jenkins please retry a build

**Pull request merged**